src/kernels/MIOpenLRNBwd.cl

```diff
@@ -161,6 +161,7 @@ MIOpenLRNWithinChannelBwd(const __global _FLOAT* top,
         }
     }
 
+    barrier(CLK_LOCAL_MEM_FENCE);
```

Comment:
- Do we need this fix for wavesize32 devices (Navi21) only?
- If we apply it to _all_ targets, does it affect performance?